.circleci/config.yml

```
@@ -422,6 +416,23 @@ jobs:
           background: true
           paths:
             - ./habitat-api
+      - run:
+          name: Build api documentation
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat; cd habitat-api
+              python setup.py develop --all
+              
+              cd docs
+              conda install -y -c conda-forge doxygen==1.8.16
+              conda install -y  jinja2 pygments docutils
+              mkdir -p ../build/docs
+              ./build-public.sh
+      - save_cache:
+          key: docs-{{ .Branch }}-{{ .Environment.CIRCLE_SHA1 }}
+          background: true
+          paths:
+            - ./habitat-sim/build/docs-public
```

API docs were missing from the cache:
```suggestion
            - ./habitat-sim/build/docs-public
            - ./habitat-api/docs/build/docs-public
```